# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../synaptics/amd64-libdir.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -Naur synaptics-0.14.2.orig/Makefile synaptics-0.14.2/Makefile
--- synaptics-0.14.2.orig/Makefile	2005-05-17 00:40:14.000000000 +0300
+++ synaptics-0.14.2/Makefile	2005-11-30 11:39:03.000000000 +0200
@@ -16,7 +16,7 @@
 endif
 ifeq ($(ARCH),x86_64)
   ARCH_DEFINES = -D__x86_64__ -D_XSERVER64
-  LIBDIR = lib64
+  LIBDIR = lib
 else
   ARCH_DEFINES = -D__i386__
   LIBDIR = lib
